src/main/java/com/example/bookstoreproject/api/book/BookDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.example.bookstoreproject.api.book;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+@Getter
+@Builder
+public class BookDTO {
+    private UUID id;
+    private String title;
+    private String author;
+    private String description;
+    private Timestamp createAt;
+    private Timestamp updateAt;
```

Comment:
Use Instant instead of Timestamp for this one and all

Reply:
DONE